internal/service/wafv2/rule_group.go

```diff
@@ -98,9 +100,21 @@ func resourceRuleGroup() *schema.Resource {
 						validation.StringMatch(regexache.MustCompile(`^[0-9A-Za-z_-]+$`), "must contain only alphanumeric hyphen and underscore characters"),
 					),
 				},
+				"rule_json": {
```

Comment:
Actually, this _should_ be `rules_json`. The structured equivalent (`rule`) follows our naming rule of a singular for a repeated block.